src/AppContext.js

```diff
@@ -1,16 +1,23 @@
-import { createContext } from 'react';
+// @flow
+
+import { createContext, type Context } from 'react';
+import { type CategoryLookupTables } from './lib/Categories';
 
 // Only extend this with value that are sensible global values that potentially might be needed
 // everywhere
 export type AppContext = {
   baseUrl: string,
+  categories: CategoryLookupTables,
 };
 
-// FIXME
-// The next line should be rather:
-// const { Provider, Consumer } = createContext<AppContext>();
-//
-// Adding the correct generic typing here produces errors during compilation though.
-const { Provider, Consumer } = createContext();
+const { Provider, Consumer } = createContext<AppContext>({
```

Comment:
+2: Wonderful. Thx for fixing this. I tried this before and it didn't work. Now my suspect is that I forgot the `// @flow` line at the top.

Reply:
Yes, thats it.